packages/mattermost-redux/src/selectors/entities/threads.ts

```diff
@@ -182,3 +182,17 @@ export const getThreadsInChannel: (
         return Object.keys(allThreads).filter((id) => allThreads[id].post.channel_id === channelID);
     },
 );
+
+export const getThreadItemsInChannel: (
```

Comment:
nit
Can we move this to 'selectors/views/threads.ts' instead of it being in MM

Reply:
atleast for the new code, which is tied to only webapp can be moved away from mm-redux

Reply:
@M-ZubairAhmed this selector is being used in an action for counts, which lives in `mattermost-redux` package, that's why it's not moved (sorry I missed that before).

Reply:
ahh got it